llvm/utils/lit/tests/Inputs/shtest-cat/check_path.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
Instead of adding this file, can we move the existing copy up a directory?that is probably it’s own patch.